pkgs/tools/filesystems/nixpart/0.4/multipath-tools.nix

```diff
@@ -10,11 +10,11 @@ stdenv.mkDerivation rec {
 
   sourceRoot = ".";
 
-  buildInputs = [ lvm2 libaio readline ];
+  buildInputs = [ lvm2 libaio readline gzip ];
 
   preBuild =
     ''
-      makeFlagsArray=(GZIP="${gzip}/bin/gzip -9 -c" prefix=$out mandir=$out/share/man/man8 man5dir=$out/share/man/man5 LIB=lib)
+      makeFlagsArray=(GZIP="-9" prefix=$out mandir=$out/share/man/man8 man5dir=$out/share/man/man5 LIB=lib)
```

Comment:
This looks a bit weird, does this still build the manpages properly? I'm just asking out of curiosity, because even if the manpages fail to compress it doesn't matter much because it's solely for the bootstrapper.